tools/CheckHelp.ps1

```diff
@@ -84,12 +124,176 @@ Get-Command -Type Cmdlet -Module PSReadLine |
+#region "Check if bindable functions are all with descriptions"
+
 Get-PSReadLineKeyHandler -Bound -Unbound |
     Where-Object { $_.Function -eq $_.Description } |
     ForEach-Object {
         ReportError "Function missing description: $($_.Function)"
     }
 
+#endregion
+
+#region "Check if default key bindings are all documented"
+
+class BindableFunction
+{
+    [string]$Function
+    [string]$Description
+    [string[]]$CmdBindings = @()
+    [string[]]$EmacsBindings = @()
+    [string[]]$ViInsBindings = @()
+    [string[]]$ViCmdBindings = @()
+    # Can't use the following type because it's not loaded at parse time
+    <#[Microsoft.PowerShell.KeyHandlerGroup]#>[object]$Group
+}
+
+function AppendTextWithLineWrapping
+{
+    param(
+        [int] $Column,
+        [string] $EndOfLine,
+        [string] $LineToAppend,
+        [System.Text.StringBuilder] $Buffer
+    )
+
+    $maxWidth = 80
+    $indent = "    "
+    if ($LineToAppend + $Column -lt $maxWidth)
+    {
+        $Buffer.Append($LineToAppend)
+    }
+    else
+    {
+        # Wrap the text
+        $col = $Column
+        $words = $LineToAppend -split ' '
+        foreach ($word in $words)
+        {
+            if ($col + $word.Length -ge $maxWidth)
+            {
+                if ($Buffer[$Buffer.Length - 1] -eq " ")
+                {
+                    $Buffer.Remove($Buffer.Length - 1, 1)
+                }
+
+                $Buffer.Append("${EndOfLine}${indent}")
+                $col = $indent.Length
+            }
+
+            $col += ($word.Length + 1)
+            $Buffer.Append($word)
+            $Buffer.Append(" ")
+        }
+
+        $Buffer.Remove($Buffer.Length - 1, 1)
+    }
+}
+
+Set-PSReadLineOption -EditMode Windows
+$cmdKeyBindings = Get-PSReadLineKeyHandler -Bound
+
+Set-PSReadLineOption -EditMode Emacs
+$emacsKeyBindings = Get-PSReadLineKeyHandler -Bound
+
+Set-PSReadLineOption -EditMode Vi
+$viKeyBindings = Get-PSReadLineKeyHandler -Bound
+
+$bindableFunctionsFromReflection = ((Get-Command Set-PSReadLineKeyHandler).Parameters['Function'].Attributes).ValidValues
+$allFunctions = @{}
+
+foreach ($fn in $bindableFunctionsFromReflection)
+{
+    $bindableFn = [BindableFunction]::new()
+    $bindableFn.Function = $fn
+    $bindableFn.Group = [Microsoft.PowerShell.PSConsoleReadLine]::GetDisplayGrouping($fn)
+
+    $allFunctions[$fn] = $bindableFn
+}
+
+foreach ($binding in $cmdKeyBindings)
+{
+    $allFunctions[$binding.Function].CmdBindings += $binding.Key
+}
+
+foreach ($binding in $emacsKeyBindings)
+{
+    $allFunctions[$binding.Function].EmacsBindings += $binding.Key
+}
+
+foreach ($binding in $viKeyBindings)
+{
+    $mode = if ($binding.Key[0] -eq '<') { 'ViCmdBindings' } else { 'ViInsBindings' }
+    # Some vi functions are private but are stil reported by Get-PSReadLineKeyHandler
+    $bindableFn = $allFunctions[$binding.Function]
+    if ($null -ne $bindableFn)
+    {
+        $bindableFn.$mode += $binding.Key
+    }
+}
+
+$eol = "`r`n"
```

Comment:
It seems a little silly to explicitly use `` `r`n `` when `` `n `` works too. Or you could use `[Environment]::NewLine` to make it "obvious" that it works cross platform even though as written it seems like should work on Linux anyway.

Reply:
I use CRLF explicitly here because the 'about_PSReadLine.help.txt' pulled down by 'Update-Help' uses CRLF end-of-line sequence, on both Windows and Unix plats.
If switching to LF only, the wildcard matching all always fail. I added a comment there.